cloudprober/templates/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          args: ['--config_file', '/cfg/cloudprober.{{- .Values.configFormat -}}', '--logtostderr']
+          args: ['--config_file', '/cfg/{{- .Values.configFileName | default "cloudprober" -}}.{{- .Values.configFormat -}}', '--logtostderr']
```

Comment:
See my comment above about keeping default value in values.yaml. Also while here can you just remove --logtostderr.